### ISSUE TITLE
Mail crashes everytime I try to send an e-mail - BUG

### ISSUE THREAD

**Issue by Aglais1** [UniversalMailer/UniversalMailer#102]
Hello,
after installing the plugin I am unable to send any e-mails because Mail crashes everytime I click the send button.
Unfortunately, I cannot remove the plugin by the help of the Mail Plugin Manager. If I try to disable the plugin, it shows the error message: "Couldn’t create the Disabled Bundle folder at “/Library/Mail/Bundles (Disabled)”.
You don’t have permission to save the file “Bundles (Disabled)” in the folder “Mail”.."
And if I try to delete the plugin, it asks me for my passwort, I type it in, click ok, and then nothing happens.

So my problems have become worse instead of better. I hope, someone can help me on that soon.
I use Mac OSX 10.11.6 and Universal Mail 3.0 (Build 16).

**Comment by Aglais1** [comment]
One thing that worked: after disabling UniversalMailer in the Mail preferences and restarting Mail, I am now able to send mails.
Still, I cannot use UniversalMailer.

**Comment by SebStar** [comment]
This is not a UM Bug..
To me it sounds more like a permission error.

Try the following:
-remove "Mail Plugin Manager"

 -remove:
~/Library/Mail/Bundles
and
/Library/Mail/Bundles

-Check your permissions for ~/Library/Mail/ (not /Library/Mail/) 
You should have full read/write access this folder and to all folders in there.

-Reinstall UM

try if it works now before you install more Mail-plugins or "Mail Plugin Manager"

Cheers

**Comment by Aglais1** [comment]
Hello,
thanks for your help. It seems to work fine now.
I removed the Mail Plugin Manager as well as the folder UM in the folder /Library/Mail/Bundles. In ~/Library/Mail was no folder called "Bundles".
Still, I reinstalled UM and now Mail doesn't crash anymore when sending mails.

By the way: I installed Mail Plugin Manager after installing UM because the UM FAQ-page told me so, if I want to remove UM.
